understanding/21/non-text-contrast.html

```diff
@@ -82,7 +82,7 @@ <h4 id="related-scs">Relationship with Use of Color and Focus Visible</h4>
 				<figure id="figure-two-star-ratings-failing">
 					<img src="img/star-examples-fail.png" alt="Two star ratings, the first uses 5 stars with a black outline and a yellow or white fill, where yellow indicates checked. The second uses only pale yellow stars on white." width="300" />
 					<figcaption>
-						Two examples which fail a success criteria, the first fails the Use of color criteria due to relying on yellow and white hues. The second example fails the Non-text contrast criteria due to the yellow (#fff000) to white contrast ratio of 1.2:1.
+						Two examples which fail a success criteria, the first fails the Use of color criteria due to relying on yellow and white hues. The second example fails the Non-text contrast criteria due to the yellow (#fff000) to white background of 1.2:1.
```

Comment:
the sentence now reads very awkwardly without the "contrast ratio" to me. should it perhaps be "the yellow (#fff000) to white background contrast ratio of 1.2:1"?

or turn it around and expand it as "fails the Non-text contrast criterion due to the low contrast ratio of 1.2:1 of the yellow (#fff000) against the white background."

Reply:
additionally, while we're making edits...there seem to be - once again - instances in this document that use "success criteria" (plural) when they mean "success criterion" (singular).

taking the above second suggestion, and fixing up the start of the sentence as well, it should probably be more:

"Two examples which fail two different success criteria: the first fails the Use of color criterion due to relying on yellow and white hues; the second example fails the Non-text contrast criterion due to the low contrast ratio of 1.2:1 of the yellow (#fff000) against the white background."

Reply:
Hey

We agreed in today's AGWG call that there's a little more editing to be done on that "two examples" update, but I was asked to push the wording as is and @mbgower and @gundulaniemann were going to work on improving in.

Thanks for the feedback.

Reply:
Hello, 
I understand one point of irritation was naming the hues without pointing out that the contrast does not contribute to the distinction.
As for my impression the most recent suggested version reads very fluently and clearly, I picked it up to suggest the following.

"Two examples which fail two different success criteria: the first fails the Use of color criterion due to relying on variation in color alone (yellow and white hues, without adequate contrast); the second example fails the Non-text contrast criterion due to the low contrast ratio of 1.2:1 of the yellow (#fff000) against the white background."